### PR TITLE
Update Step Functions troubleshooting doc for missing TaskScheduled event

### PR DESCRIPTION
**Pull request by kimi-p**
### What does this PR do? What is the motivation?
Help Step Functions customer to grade to recommended Lambda integrations if they are still using legacy Lambda API/Integrations.
https://docs.google.com/document/d/1QeCpsVt9a-zT2ypIDve7FXUTSeojS-JAwMMNDyRC038/edit

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by nine5two7 — APPROVED** [review]
lgtm

File: content/en/serverless/step_functions/troubleshooting.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ If your organization has an existing all-encompassing index with a low limit, pl
 - Verify that you can see both Lambda traces and Step Function traces in Datadog.
 - Verify that you are using Python layer v75+ or Node.js layer v94+.
 - In your AWS console, open your Step Function and ensure that your state machine has `"Payload.$": "States.JsonMerge($$, $, false)"` on the Lambda steps.
-- Execute your Step Function once and verify that the `TaskScheduled` event log of the Lambda step has the payload containing data from the [Step Function context object][3].
+- Execute your Step Function once and verify that the `TaskScheduled` event log of the Lambda step has the payload containing data from the [Step Function context object][4]. If you do not have `TaskScheduled` event and only have `LambdaFunctionScheduled` event, please update the task in Step Functions definition to use the recommended Lambda integration by following the [AWS documentation][5].
```
Comment (cswatt) [inline content/en/serverless/step_functions/troubleshooting.md:38]:
```suggestion
- Execute your Step Function once and verify that the `TaskScheduled` event log of the Lambda step has the payload containing data from the [Step Function context object][4]. If you do not have a `TaskScheduled` event and only have a `LambdaFunctionScheduled` event, update the task in Step Functions definition to use the recommended Lambda integration. See the [AWS documentation][5] for instructions on how to do this.
```

**Review by cswatt — APPROVED** [review]

File: content/en/serverless/step_functions/troubleshooting.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ If your organization has an existing all-encompassing index with a low limit, pl
 - Verify that you can see both Lambda traces and Step Function traces in Datadog.
 - Verify that you are using Python layer v75+ or Node.js layer v94+.
```
Comment (nine5two7) [inline content/en/serverless/step_functions/troubleshooting.md:36]: @kimi-p  Should me also update the Python and Node.js layer version for the 128 bits change?

Comment (kimi-p) [reply content/en/serverless/step_functions/troubleshooting.md:36]: This is done in the previous PR https://github.com/DataDog/documentation/pull/23546

**Review by cswatt — APPROVED** [review]

**Pull request merged**